DirectXTex/DirectXTexConvert.cpp

```diff
@@ -2998,8 +2997,8 @@ uint32_t DirectX::_GetConvertFlags(DXGI_FORMAT format) noexcept
 #endif
 
     ConvertData key = { format, 0, 0 };
-    auto in = reinterpret_cast<const ConvertData*>(bsearch_s(&key, g_ConvertTable, std::size(g_ConvertTable), sizeof(ConvertData),
-        ConvertCompare, nullptr));
+    auto in = reinterpret_cast<const ConvertData*>(bsearch(&key, g_ConvertTable, std::size(g_ConvertTable), sizeof(ConvertData),
```

Comment:
``bsearch_s`` like most of the ``_s`` variants in the C++17 Appendex K are not supported by GNUC's library. As it turns out, in this case it doesn't matter at all. The difference with ``bsearch_s`` is that it passes a context pointer through, which we don't use here anyhow. Just simplified it to always use ``bsearch``.